### DISCUSSION TITLE
Expand Docker Address Pool

### DISCUSSION THREAD

**Discussion by hunterwinterton** [Jip-Hop/jailmaker#200] [General]
Hey, I'm wondering if/how I can expand the address pool to allow docker to use more ip addresses. I have run out and was wondering if theres a way to expand the docker address pool within Jailmaker. 
I found I could do something like this in /etc/docker/daemon.json, but I couldn't find the file and didn't know if it would persist if I created it. Is this the way to do it, or is there another setting? Thanks for your help!

{
  "default-address-pools": [
    {
      "base": "172.80.0.0/16",
      "size": 24
    },
    {
      "base": "192.168.0.0/16",
      "size": 24
    }
  ]
}

**Comment by Jip-Hop** [comment]
Just try it ;)
